crates/rpc/rpc/src/debug.rs

```diff
@@ -546,9 +554,10 @@ where
 
                         let (res, _) = inspect(db, env, &mut inspector)?;
 
-                        let frame = inspector
-                            .into_geth_builder()
-                            .geth_call_traces(call_config, res.result.gas_used());
+                        let geth_builder = inspector.into_geth_builder();
+                        let frame =
+                            geth_builder.geth_call_traces(call_config, res.result.gas_used());
+                        self.drop_in_task(geth_builder);
```

Comment:
with #7063 

the call+prestate tracer lo longer record any stack+memory so I think we can get away without handling these